### ISSUE TITLE
Please make apps-config public

### ISSUE THREAD

**Issue by vikiival** [polkadot-js/apps#2537]
Hello, 

I would like to use package `apps-config` along with [Vue Polkadot](https://github.com/vue-polkadot)

I need that package for api and setting.

What's the problem: 

on npm install: 
```bash
npm install @polkadot/apps-config --save
```
I get:

```bash
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@polkadot%2fapps-config - Not found
npm ERR! 404 
npm ERR! 404  '@polkadot/apps-config@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
```

I expect that `npm install @polkadot/apps-config --save` would be installed successfully.

**Comment by yangwao** [comment]
Can confirm
![image](https://user-images.githubusercontent.com/5887929/78870477-78ba5c00-7a46-11ea-8fc0-25c6c1a9e0dc.png)

Package is private now, but why?
https://github.com/polkadot-js/apps/blob/master/packages/apps-config/package.json#L3

**Comment by jacogr** [comment]
These are not meant to be published, is is config for the `apps` package, in the same way as `apps-routing` is config for the paths.

The non-published packages (which here is effectively everything) is not in a state for external consumption. If they were, they would be in in the `ui` repo. It makes no sense publishing these (was there, got burned) since it just adds overhead to CI for no benefit.

As soon as one get published, it means more need to get published (e.g. config also has links to the components, which is React-only).

**Comment by jacogr** [comment]
Ok, looked though, and there are no React-specific deps, which is good.

Sadly, doesn't change the outcome - e.g. in the above you wanted to have a "stable version" (that package would not have been published as "stable" before today - todays' 0.40 was only done because the beta numbering is >200, the versions here have no effect), so "stable" are few and far in-between. 

With the number of merges going in, we are really not going back to publishing all on CI again here, it adds 2x the current turn-around, with no benefit apart from additional support overhead.
